cli/tests/mysql.spec.ts

```diff
@@ -0,0 +1,287 @@
+import { test } from '@playwright/test';
+import * as cli from '@helpers/cliHelper';
+
+const MYSQL_USER = 'msandbox'
+const MYSQL_PASSWORD = "msandbox"
+
+test.describe('PMM Client CLI tests for MySQL', async () => {
+
+  /**
+   * @link https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats#L29
+   */
+  test('run pmm-admin add mysql based on running instances', async ({ }) => {
+    let hosts = (await cli.exec(`sudo pmm-admin list | grep "MySQL" | awk -F" " '{print $3}'`))
+      .stdout.trim().split('\n').filter((item) => item.trim().length > 0);
+    let n = 1;
+    for (const host of hosts) {
+      let output = await cli.exec(`sudo pmm-admin add mysql --query-source=perfschema --username=${MYSQL_USER} --password=${MYSQL_PASSWORD} mysql_${n++} ${host}`);
+      await output.assertSuccess();
+      await output.outContains('MySQL Service added.');
+    }
+  });
+
+  /**
+   * @link https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats#L44
+   */
+  test('run pmm-admin add mysql again based on running instances', async ({ }) => {
+    let hosts = (await cli.exec(`sudo pmm-admin list | grep "MySQL" | grep "mysql_" | awk -F" " '{print $3}'`))
+      .stdout.trim().split('\n').filter((item) => item.trim().length > 0);
+    let n = 1;
+    for (const host of hosts) {
+      let output = await cli.exec(`sudo pmm-admin add mysql --query-source=perfschema --username=${MYSQL_USER} --password=${MYSQL_PASSWORD} mysql_${n++} ${host}`);
+      await output.exitCodeEquals(1);
+      await output.outContains('already exists.');
+    }
+  });
+
+  /**
+  * @link https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats#L58
+ */
+  test('run pmm-admin remove mysql added using current running instances', async ({ }) => {
+    let hosts = (await cli.exec(`sudo pmm-admin list | grep "MySQL" | grep "mysql_"`))
+      .stdout.trim().split('\n').filter((item) => item.trim().length > 0);
+    let n = 1;
+    for (const host of hosts) {
+      let output = await cli.exec(`sudo pmm-admin remove mysql mysql_${n++}`);
+      await output.assertSuccess();
+      await output.outContains('Service removed.');
+    }
+  });
+
+  /**
+   * @link https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats#L77
+   */
+  test('run pmm-admin add mysql --help to check metrics-mode=auto', async ({ }) => {
+    const output = await cli.exec('sudo pmm-admin add mysql --help');
+    await output.assertSuccess();
+    await output.outContains(`metrics-mode=\"auto\"`);
+  });
+
+  /**
+   * @link https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats#L162
+   */  
+  test('run pmm-admin add mysql --help to check socket', async ({ }) => {
+    const output = await cli.exec('sudo pmm-admin add mysql --help');
+    await output.assertSuccess();
+    await output.outContains('socket=STRING');
```

Comment:
Looks like the same check is present at line 81. Probably given test can be removed since the check is covered by the following test.